Sources/MessagingPush/PushClickHandler.swift

```diff
@@ -170,6 +180,25 @@ extension PushClickHandlerImpl {
         // continue swizzle
         cio_swizzle_didReceive(center, didReceive: response, withCompletionHandler: completionHandler)
     }
+
+    // Swizzled method that gets called when a push notification gets clicked or swiped away
```

Comment:
This is a swizzled method for `willPresentNotification` that basically means that this method will be called only if the application is in the foreground and is used to present the push with a sound, badge, alert etc. Reading the comment confuses me as how does this method gets called when a push notification gets clicked or swiped away - interaction??? 🤔

Reply:
```suggestion
    // Swizzled method that gets called before the OS displays the push. Used to determine if a push gets displayed while app is in foreground or not. 
```